vms/avm/service_test.go

```diff
@@ -845,30 +820,16 @@ func TestServiceGetTxJSON_OperationTxWithNftxMintOp(t *testing.T) {
 
 	key := keys[0]
 	createAssetTx := newAvaxCreateAssetTxWithOutputs(t, vm)
-	_, err := vm.IssueTx(createAssetTx.Bytes())
-	require.NoError(err)
+	issueAndAccept(require, vm, issuer, createAssetTx)
 
 	mintNFTTx := buildOperationTxWithOp(buildNFTxMintOp(createAssetTx, key, 2, 1))
 	require.NoError(mintNFTTx.SignNFTFx(vm.parser.Codec(), [][]*secp256k1.PrivateKey{{key}}))
-
-	txID, err := vm.IssueTx(mintNFTTx.Bytes())
-	require.NoError(err)
-	require.Equal(mintNFTTx.ID(), txID)
-	ctx.Lock.Unlock()
-
-	msg := <-issuer
-	require.Equal(common.PendingTxs, msg)
-	ctx.Lock.Lock()
-
-	txs := vm.PendingTxs(context.Background())
-	require.Len(txs, 2)
-	require.NoError(txs[0].Accept(context.Background()))
-	require.NoError(txs[1].Accept(context.Background()))
```

Comment:
A number of the existing tests expected to be able to verify transactions based on processing state... This no longer works so now rather than:

`IssueTx` -> `IssueTx` -> `Accept` -> `Accept`

we do:

`IssueTx` -> `Accept` -> `IssueTx` -> `Accept`